ui/src/treeDataProvider/AzureTreeDataProvider.ts

```diff
@@ -249,10 +249,10 @@ export class AzureTreeDataProvider implements TreeDataProvider<IAzureNode>, Disp
             nodes = [new AzureNode(undefined, { label: commandLabel, commandId: 'azure-account.selectSubscriptions', contextValue: 'azureCommandNode', id: 'azure-account.selectSubscriptions' })];
```

Comment:
How about some comments, name changes, or property getters to make this clearer to everyone?